### PR TITLE
fix: headers, license

### PR DESCRIPTION
**Pull request by Irene-123**
Hello there, 
I have updated the readme by fixing the headers, adding a license tag and some points. The changes are quite basic, if we need to do any other major changes, do let me know. Thank you

### CODE REVIEW

**Comment by yixinBC** [comment]
nice job!

File: README.md [hunk 1]
```diff
@@ -55,20 +56,27 @@ cask "chatgpt", args: { "no-quarantine": true }
 
 ## 📢 Announcement
 
+### ChatGPT Prompts!
 This is a major and exciting update. It works like a `Telegram bot command` and helps you quickly populate custom models to make chatgpt work the way you want it to. This project has taken a lot of my spare time, so if it helps you, please help spread the word or star it would be a great encouragement to me. I hope I can keep updating it and adding more interesting features.
 
 ### How does it work?
 
-You can look at [awesome-chatgpt-prompts](https://github.com/f/awesome-chatgpt-prompts) to find interesting features to import into the app.
+You can look at <b>[awesome-chatgpt-prompts](https://github.com/f/awesome-chatgpt-prompts)</b> to find interesting features to import into the app.
```
Comment (yixinBC) [inline README.md:64]: It's better to use \*\*  \*\* for bold

File: README.md [hunk 2]
```diff
@@ -172,19 +180,19 @@ Currently, only json and csv are supported for synchronizing custom files, and t
 
 ## ❓FAQ
 
-### Can't open ChatGPT
+### 1. Can't open ChatGPT
```
Comment (yixinBC) [inline README.md:183]: those four elements are in one level and don't have particular orders.

File: README.md [hunk 3]
```diff
@@ -172,19 +180,19 @@ Currently, only json and csv are supported for synchronizing custom files, and t
...
 
 If you cannot open the application after the upgrade, please try to clear the configuration file, which is in the `~/.chatgpt/*` directory.
 
-### Out of sync login status between multiple windows
+### 2. Out of sync login status between multiple windows
```
Comment (yixinBC) [inline README.md:187]: those four elements are in one level and don't have particular orders.

File: README.md [hunk 4]
```diff
@@ -172,19 +180,19 @@ Currently, only json and csv are supported for synchronizing custom files, and t
...
 
 If you have already logged in in the main window, but the system tray window shows that you are not logged in, you can fix it by restarting the application (`Menu -> Preferences -> Restart ChatGPT`).
 
-### Is it safe?
+### 3. Is it safe?
```
Comment (yixinBC) [inline README.md:191]: those four elements are in one level and don't have particular orders.

File: README.md [hunk 5]
```diff
@@ -172,19 +180,19 @@ Currently, only json and csv are supported for synchronizing custom files, and t
...
 
 It's safe, just a wrapper for [OpenAI ChatGPT](https://chat.openai.com) website, no other data transfer exists (you can check the source code).
 
-### Developer cannot be verified?
+### 4. Developer cannot be verified?
```
Comment (yixinBC) [inline README.md:195]: those four elements are in one level and don't have particular orders.

**Review by yixinBC — CHANGES_REQUESTED** [review]

**Comment by yixinBC** [comment]
but there are still some improvements

File: README.md (hunk 2 above)
Comment (yixinBC) [reply README.md:183]: so just don't make changes there

**Comment by Irene-123** [comment]
> but there are still some improvements

Tell me some rough improvements, I'll add it

**Comment by yixinBC** [comment]
> > but there are still some improvements
> 
> Tell me some rough improvements, I'll add it

look through my code reviews above.

File: README.md (hunk 3 above)
Comment (Irene-123) [reply README.md:187]: sure

**Pull request merged**